schemas/code_signature.yml

```
@@ -0,0 +1,59 @@
+---
+- name: code_signature
+  title: Code Signature
+  group: 2
+  description: These fields contain information about binary code signatures.
```

What about a shorter short definition, and a more fleshed out full definition?

```suggestion
  description: These fields contain information about code signatures for various forms of executable code, such as executables, libraries or scripts.
  short: Information about binary code signatures.
```

And the above begs the question: will we be wanting to track signatures other than code signatures? If so, do we want to discuss whether this field set should be named `signature` instead? Happy to close that can of worms immediately, if you think it's an unlikely hypothetical ;-)

---

i thought about that, and the big downside I came up with is that the word "signature" is often used like "rule", so I didn't want to potentially clobber up a namespace that may be used.

I definitely want something shorter than `code_signature`, because `process.code_signature.valid` is lengthy

---

I think we should stick with `code_signature`, then. Are you good with that?